deployments/README.md

```diff
@@ -121,7 +121,9 @@ Following accounts are used when `rialto-millau` bridge is running:
 - Millau's `Eve` signs relay transactions with message delivery confirmations (lane 00000001) from Rialto to Millau;
 - Rialto's `Eve` signs relay transactions with messages (lane 00000001) from Millau to Rialto;
 - Millau's `Ferdie` signs relay transactions with messages (lane 00000001) from Rialto to Millau;
-- Rialto's `Ferdie` signs relay transactions with message delivery confirmations (lane 00000001) from Millau to Rialto.
+- Rialto's `Ferdie` signs relay transactions with message delivery confirmations (lane 00000001) from Millau to Rialto;
+- Millau's `RialtoMessagesOwner` signs relay transactions with updated Rialto -> Millau conversion rate;
+- Rialto's `MillauMessagesOwner` signs relay transactions with updated Millau -> Rialto conversion rate.
```

Comment:
We are using same accounts, but I personally had many problems when was trying to select 'vacant' account or when trying to understand what's given account is doing in our deployments. Hence, this section in readme. I would prefer dev accounts in our test chains with meaningful names, but there's code that we're using (substrate cli) that depends on standard names (because it parses cli options like `--alice`, ... and we don't want to start extra nodes). So probably it's a good moment to start using some meaningful names - i.e. `RialtoMessagesOwner` is an owner of with-Rialto messages pallet at Millau? If you're not convinced, I could add `Ingrid` instead :)

Reply:
Custom names sounds good, I was just wondering if it's worth to have different accounts for Rialto and Millau, but I guess we might catch some subtle mistakes here and there thanks to this.